Market-API.txt

```diff
@@ -9,7 +9,13 @@ The current on that is:
 
 from tribler/Tribler/community/market/community.py
 
-Division of responsibility: the market is fully responsible for the exchange of funds and management of the wallets of both Multi-chain, Bitcoin.
+Division of responsibility: 
+The Multi-chain database, wallet and money transfer mechanism is fully under control of The Market team.
+A get_balance() function for multi-chain needs to be created by either: one of the BEP teams of dev team of multi-chain.
+
+Both project require access and control of the Bitcoin wallet.
+A cmd line gateway will be used to get_balance(), create_transaction() and verify_payment().
+This gateway will be used by both BEP projects.
 
```

Comment:
to the other bep group: this already exists in the current codebase of autonomous self-replicating code. we could put it in a seperate repository